src/portal/views/Twin.vue

```diff
@@ -93,7 +93,7 @@ export default class TwinView extends Vue {
     this.accountName = this.$credentials.accountName;
   }
   mounted() {
-    if (this.$api && this.$credentials) {
+    if (this.$api && this.$credentials.twinIP !== "" && this.$credentials.twinID != 0) {
```

Comment:
I think you still need to check if `this.$credentials` is defined before accessing the twinIp on it, so you don't get an error while loading the page